src/Files.App/Views/MainPage.xaml.cs

```diff
@@ -198,23 +203,23 @@ private void PaneHolder_PropertyChanged(object? sender, PropertyChangedEventArgs
 			UpdateNavToolbarProperties();
 			LoadPaneChanged();
 		}
-
+		// SidebarAdaptiveViewModel.PaneHolder must not be null
 		private void UpdateStatusBarProperties()
 		{
 			if (StatusBarControl is not null)
 			{
 				StatusBarControl.DirectoryPropertiesViewModel = SidebarAdaptiveViewModel.PaneHolder?.ActivePaneOrColumn.SlimContentPage?.DirectoryPropertiesViewModel;
-				StatusBarControl.SelectedItemsPropertiesViewModel = SidebarAdaptiveViewModel.PaneHolder?.ActivePaneOrColumn.SlimContentPage?.SelectedItemsPropertiesViewModel;
+				StatusBarControl.SelectedItemsPropertiesViewModel = SidebarAdaptiveViewModel.PaneHolder!.ActivePaneOrColumn.SlimContentPage.SelectedItemsPropertiesViewModel; 
 			}
 		}
-
+		// SidebarAdaptiveViewModel.PaneHolder must not be null
```

Comment:
Add a new line before this line.

According to the guideline Section 3, Item 4.
> Insert a blank line after closing curly bracket.